pkg/defaultinvariants/types.go

```diff
@@ -65,6 +66,8 @@ func newDisruptiveInvariants() invariants.InvariantRegistry {
 
 	invariantTests.AddInvariantOrDie("apiserver-availability", "kube-apiserver", disruptionlegacyapiservers.NewRecordAvailabilityOnly())
 
+	invariantTests.AddInvariantOrDie(disruptionpodnetwork.InvariantName, disruptionpodnetwork.JIRAOwner, disruptionpodnetwork.NewPodNetworkAvalibilityInvariant())
```

Comment:
This looks like it should be under `newDefaultInvariants` a few lines up.  Disruptive is like disaster recovery.  Upgrades are non-disruptive.